x/rollup/tests/integration/rollup_test.go

```diff
@@ -50,9 +50,12 @@ func TestRollup(t *testing.T) {
 	mintAmount := depositTx.Mint()
 	transferAmount := depositTx.Value()
 	from, err := gethtypes.NewCancunSigner(depositTx.ChainId()).Sender(depositTx)
+
+	require.NoError(t, err)
```

Comment:
nit: add `require.NoError(t, err)` directly below `err` without a newline in between